### PR TITLE
fix vae with scipy issue

### PR DESCRIPTION
**Pull request by glorysdj**
try to fix https://github.com/intel-analytics/analytics-zoo/issues/2187

---
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/2197)

### CODE REVIEW

**Comment by glorysdj** [comment]
fixed 
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Python-AppTests-part2/11/console

File: apps/variational-autoencoder/utils.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import scipy.misc
```
Comment (hkvision) [inline apps/variational-autoencoder/utils.py:1]: So do we still need scipy now?

**Comment by yangw1234** [comment]
LGTM

**Comment by hkvision** [comment]
Then why not remove scipy in the code???

**Comment by glorysdj** [comment]
@Kai, sorry, missed your comments, i will remove import scipy.misc.

**Comment by hkvision** [comment]
Need to remove the import in these three files.

**Pull request merged**